### PR TITLE
Avishay/mgmt 848 v2

### PR DESCRIPTION
**Pull request by avishayt**

### CODE REVIEW

**Comment by avishayt** [comment]
@ronniel1 Does anything need to be done from this end for the trigger to push the new image to ocp-metal/s3-object-expirer:stable ?

File: Makefile [hunk 1]
```diff
@@ -69,6 +74,11 @@ deploy-service-requirements: deploy-inventory-service-file
 deploy-service: deploy-service-requirements deploy-role
 	python3 ./tools/deploy_assisted_installer.py
 
+deploy-expirer: deploy-role
+	sed "s#REPLACE_IMAGE#${OBJEXP}#g" deploy/s3/s3-object-expirer-cron.yaml > deploy/s3/s3-object-expirer-cron-tmp.yaml
```
Comment (filanov) [inline Makefile:78]: probably should be a python script. we moved all the deployments to python. you can see example under `tools`

File: README.md [hunk 2]
```diff
@@ -20,6 +20,11 @@ To push your build target to a docker registry you first need to change the defa
 export SERVICE=quay.io/<username>/bm-inventory:<tag>
 ```
 
+Do the same for s3-object-expirer:
+```shell script
+export SERVICE=quay.io/<username>/s3-object-expirer:<tag>
```
Comment (filanov) [inline README.md:25]: `OBJEXP=...`

File: deploy/s3/s3-object-expirer-cron.yaml [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+          - name: s3-object-expirer-job-pod
+            image: REPLACE_IMAGE
+            imagePullPolicy: Always
+            envFrom:
+              - configMapRef:
+                  name: s3-config
+            args:
+            - /bin/bash
+            - -c
+            -  python ./expirer.py
```
Comment (filanov) [inline deploy/s3/s3-object-expirer-cron.yaml:21]: By default it's python 3? 
In bm-inventory the default is python2.7 this is why i ask.

File: tools/expirer.py [hunk 4]
```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/python
+
+import boto3
+from datetime import datetime, timedelta
+import os
+import pytz
+
+S3_ENDPOINT_URL = os.environ.get('S3_ENDPOINT_URL', 'http://10.35.59.36:30925')
```
Comment (filanov) [inline tools/expirer.py:8]: not sure that there need to a default.

**Comment by filanov** [comment]
@ronniel1 will the action that we have generate this iso as well?

File: deploy/s3/s3-object-expirer-cron.yaml (hunk 3 above)
Comment (avishayt) [reply deploy/s3/s3-object-expirer-cron.yaml:21]: I saw Osher's scripts use 2.7, not sure why.  My Dockerfile is FROM python:3.  My script can run with either though.

**Comment by avishayt** [comment]
@ronniel1 @filanov Please take a look again.  Not sure how to test the github action...

**Comment by filanov** [comment]
@avishayt you can probably add subsystem test to validate this image works. 
not sure about the action myself.
the rest of the code LGTM

**Comment by avishayt** [comment]
@ronniel1 Looks good?

**Pull request merged**